components/brave_rewards/browser/rewards_protocol_handler.h

```diff
@@ -18,6 +18,14 @@ void HandleRewardsProtocol(const GURL& url,
                            content::WebContents::OnceGetter web_contents_getter,
                            ui::PageTransition page_transition);
 
+bool IsValidWalletProviderRedirect(const GURL& referrer_url,
+                                   const GURL& redirect_url);
+
+bool IsValidWalletProviderRedirectImpl(
```

Comment:
Now you don't need the declaration in `components/brave_rewards/browser/rewards_protocol_handler_unittest.cc`.